README.md

```diff
@@ -29,15 +29,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
-
+Map creates a new array.
 2. What is the difference between a function and a method?
-
+a method is a function inside of a function.
 3. What is closure?
-
+closure is the thing that an object needs to create a local scope.
 4. Describe the four rules of the 'this' keyword.
-
+Window binding is when this binds to something in the global scope. Implicit binding is when this is binding to something in the local scope. New binding is when this is binding to the object it has been passed to. Explicit binding is when methods are used to overwrite the implicit binding properties of this.
```

Comment:
For 'implicit binding' try to specify what the something is that it's binding to. It's totally fine if you have to go back and look a little deeper in a study session. It will make you a stronger developer in the future. :)